### ISSUE TITLE
ability to select color tables (separately for each panel of three-panel comparisons)

### ISSUE THREAD

**Issue by susburrows** [UV-CDAT/uvcmetrics#82]
it would be nice to be able to select different color tables in three panel comparisons of case A, B, and difference fields using one color table for case A and B, and a different color table for the difference plot.

**Comment by painter1** [comment]
This is one of the ways one might tinker with the graphics to improve appearance, especially for publication.  In general, that sort of thing is best done in a GUI.  I think someone else has plans to write a GUI for this purpose.

**Comment by susburrows** [comment]
I disagree -- although it might be nice to be able to look at color tables in a GUI as well, we would like to have the ability to specify the color tables within the diagnostics code, also.

But, this is a low priority issue (as you have already tagged it).

**Comment by doutriaux1** [comment]
I second @susburrows on that. But with @chaosphere2112 latest improvements we should be able to do both.
